central/scannerdefinitions/handler/handler.go

```diff
@@ -291,6 +291,12 @@ func (h *httpHandler) openDefinitions(ctx context.Context, t updaterType, opts o
 
 	online, err := h.openOnlineDefinitions(ctx, t, opts)
 	if err != nil {
```

Comment:
does this error occur upon all update failures or really just the first one (assuming no issues reading the blob store in Central)? If only the first (ie getting the initial bundle), then LGTM. If upon all updates, then we may need to rethink some things

Reply:
The updater appears to not update the 'file' previously obtained when there are errors (line `114`)

https://github.com/stackrox/stackrox/blob/26af42cdae6e012c8d28c2770ee43bb4bfcf3aa0/central/scannerdefinitions/handler/updater.go#L80-L115